docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: '3.9'
```

Comment:
nit: Te wersje zrobili deprecated https://forums.docker.com/t/docker-compose-yml-version-is-obsolete/141313